dashboard/db/schema.rb

```diff
@@ -191,6 +191,20 @@
 
   add_index "games", ["intro_video_id"], name: "index_games_on_intro_video_id", using: :btree
 
+  create_table "hint_view_requests", force: :cascade do |t|
```

Comment:
@Hamms might be worth running this feature's requirements / db plan by @philbogle if haven't yet.

The changes @mehalshah is making with callouts / videos might be a similar access pattern, though maybe lookup / split testing needs will differ for this feature (i.e., if we want to store a large # of hints in the DB to look up going forward)